server/game/cards/07-WotW/MotoChagatai.js

```diff
@@ -13,6 +13,7 @@ class MotoChagatai extends DrawCard {
         this.persistentEffect({
             condition: context => (
                 context.source.isAttacking() &&
+                context.game.currentConflict.conflictProvince.controller !== context.source.controller &&
```

Comment:
I don't think this is the right fix

Reply:
I think the better way is to update onBreakProvince to track which players have had a province broken this conflict, and then check if your current opponent has lost a province.  This should take into account a bunch of weird scenarios with Stoke, Raze to the Ground, taking control effects, etc.

Reply:
This takes into account Stoke and control effects. I have to check what Raze does.

Reply:
We can add the Raze condition as part of implementing raze.